_issues/bad_experience.md

```diff
@@ -6,18 +6,30 @@ categories: experience
   
 Q: Hillary has experience, but isn't it bad experience?  
   
-A: In the first Presidential debate, Donald Trump claimed that Hillary "has experience, but it's bad experience." This is utter nonsense. Hillary's record of public service spans four decades, starting with her first job out of law school. Here are just a few examples of [Hillary's many accomplishments](https://www.hillaryclinton.com/feed/seven-hillary-clintons-biggest-accomplishments/):
+A: In the first Presidential debate, Donald Trump claimed that Hillary "has experience, but it's bad experience." The words "bad experience" imply aa lack of success in her various roles. This is simply not true. Hillary's record of public service spans four decades, starting with her first job out of law school, and she has seen a [number of successes and accomplishments](https://www.hillaryclinton.com/feed/seven-hillary-clintons-biggest-accomplishments/) in that time:
```

Comment:
Works for me. I was working on a better response to the debate question, but honestly it's proving difficult to find articles that are primarily factual for some of these that we could site. It might take a bit more effort so switching for now is good. I'll make the change.